### PR TITLE
Add link for nightly rust and go

### PR DESCRIPTION
**Pull request by ngaut**

### CODE REVIEW

File: USAGE.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 ### System requirement
 
 + Linux or Mac OS X.
-+ Rust, nightly version is required.
++ Rust, [nightly version](https://www.rust-lang.org/downloads.html) is required.
 + Go, 1.5+ is required.
```
Comment (siddontang) [inline USAGE.md]: go need too?

**Comment by ngaut** [comment]
PTAL

**Comment by siddontang** [comment]
LGTM

**Comment by disksing** [comment]
LGTM

**Pull request merged**